distribution/argocd-applications/cert-manager.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
 spec:
   project: default
   source:
-    repoURL: https://github.com/argoflow/argoflow-aws
-    targetRevision: HEAD
+    repoURL: <<__git_repo.url__>>
```

Comment:
This setup does break the installation for people that don't change anything in the deployment and just use the default. However, it does force people to setup the repo properly. I guess setting a default in the conf file would be an good middle ground.

Reply:
Not sure we can get around this really. There is no "default installation" for integrating with AWS. You have to set your the stuff in setup.conf or nothing works

Reply:
Good point. Should probably mention this in the README and redirect people that want a quick PoC deployment or no AWS integrations to the main repo.